### PR TITLE
October and November 2018

### PR DESCRIPTION
**Pull request by jessedc**

### CODE REVIEW

**Review by asos-petergoldsmith — APPROVED** [review]

File: intro-2018-11.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+# Upcoming Events
+
+![left](assets/highres_469612543.jpeg)
+
+- Hack Night - November 14 @ Cognizant
+- Drinks Night - November 27
+- NS Breakfast - December 7
+- Google Developers Group - November 28
+- Melbourne AWS - November 28
```
Comment (bok-) [inline intro-2018-11.md:64]: We're not on for November, it conflicts with re:Invent.

File: intro-2018-11.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+^ All complaints made in any of these ways will remain confidential, be taken seriously, investigated, and dealt with appropriately.
+
+---
+
+# Upcoming Events
+
+![left](assets/highres_469612543.jpeg)
+
+- Hack Night - November 14 @ Cognizant
```
Comment (bok-) [inline intro-2018-11.md:60]: November 21 according to meetup.com

File: intro-2018-11.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+## Follow up -> #jobs
+
+---
+
+# Presentations
+
+---
+
+# Next Hack Night :computer:
+## November 14 @ Cognizant from 6:00pm
```
Comment (bok-) [inline intro-2018-11.md:136]: 21st

File: intro-2018-11.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+# Next Drinks Night :beers:
+## November 27 @ The Mill House from 6:00pm
+
+# Next NSBreakfast :egg:
+## December 7 @ Higher Ground from 7:30am
+
+---
+
+# Next Meetup 🗣
+## December 13 @ YBF Ventures from 6:00pm
```
Comment (bok-) [inline intro-2018-11.md:149]: Are we still on for this or doing a Christmas thing with GDG?

**Pull request merged**